docs/src/main/sphinx/admin/properties-query-management.rst

```diff
@@ -130,4 +149,4 @@ The :ref:`retry policy <fte-retry-policy>` to use for
 * ``NONE`` - Disable fault-tolerant execution.
 * ``TASK`` - Retry individual tasks within a query in the event of failure.
   Requires configuration of an :ref:`exchange manager <fte-exchange-manager>`.
-* ``QUERY`` - Retry the whole query in the event of failure.
\ No newline at end of file
+* ``QUERY`` - Retry the whole query in the event of failure.
```

Comment:
nit: weird diff

Reply:
auto added new line at the end of file. Will separate out a commit.